client/main.html

```diff
@@ -17,18 +17,11 @@
   {{#if currentUser}}
   <div class="formular">{{> addPlayerForm}}</div>
   <ul>
-    {{#each player}}
-    <li class="player {{selectedClass}}">{{name}}: {{score}}</li>
-
+    {{#each player in players}}
+    <li class="player {{selectedClass}}">
+      {{> PlayerCard currentPlayer=player}}
```

Comment:
The name `currentPlayer` is a bit too long for my taste. In the context of a `PlayerCard` you will have to refer to it as `this.currentPlayer`, which is redundant- a card has just one player. I would prefer something like `player`